### ISSUE TITLE
Prod: read only replica error

### ISSUE THREAD

**Issue by gregv** [opportunity-hack/evs#65]
Wasn't able to create a new account on prod, UI would respond with HTTP 500.  Seems like this is because we have two machines running as stopping one of them fixed this.

When the request is load balanced, it probably hits the machine that has mounted LiteFS in read-only mode and thus gives us this error.

![image](https://github.com/opportunity-hack/evs/assets/6913307/824c8b41-56e1-4e82-88ab-ab28f3e18b76)

Error: 
```2023-12-22T21:06:01Z app[3d8d33ef7ee089] sjc [info]GET /resources/healthcheck 200 - - 25.070 ms
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]fuse: create(): cannot create journal: read only replica
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]PrismaClientUnknownRequestError:
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]Invalid `prisma.verification.deleteMany()` invocation:
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]Error occurred during query execution:
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(SqliteFailure(Error { code: ReadOnly, extended_code: 1544 }, Some("attempt to write a readonly database"))), transient: false })
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at Rn.handleRequestError (/myapp/node_modules/@prisma/client/runtime/library.js:174:7499)
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at Rn.handleAndLogRequestError (/myapp/node_modules/@prisma/client/runtime/library.js:174:6754)
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at Rn.request (/myapp/node_modules/@prisma/client/runtime/library.js:174:6344)
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at validate2 (file:///myapp/app/routes/_auth+/signup_.verify.tsx:105:2)
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at /myapp/node_modules/@sentry/src/utils/instrumentServer.ts:180:1
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at Object.callRouteLoaderRR (/myapp/node_modules/@remix-run/server-runtime/dist/data.js:52:16)
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at callLoaderOrAction (/myapp/node_modules/@remix-run/router/router.ts:3623:16)
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at async Promise.all (index 1)
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at loadRouteData (/myapp/node_modules/@remix-run/router/router.ts:2951:19)
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]    at queryImpl (/myapp/node_modules/@remix-run/router/router.ts:2732:20) {
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]  clientVersion: '4.16.0'
2023-12-22T21:06:07Z app[3d8d333a32d089] sjc [info]}```

**Comment by gregv** [comment]
Quick fix: `% fly scale -a evs-production count 1 --max-per-region=1`
```
% fly scale show -a evs-production
VM Resources for app: evs-production

Groups
NAME	COUNT	KIND  	CPUS	MEMORY	REGIONS
app 	1    	shared	1   	256 MB	sjc
```

**Comment by gregv** [comment]
Fix for this is #83 - closing
